nucleus/admin/template/scripts/update-truststore.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -euo pipefail
+
+TRUSTSTORE_PASSWORD="changeit"
+
+echo "🔍 Detecting Java trust store path..."
+
+# Detect Java trust store path
+if [[ -f "${JAVA_HOME}/jre/lib/security/cacerts" ]]; then
+    JAVA_TRUSTSTORE="${JAVA_HOME}/jre/lib/security/cacerts"
+elif [[ -f "${JAVA_HOME}/lib/security/cacerts" ]]; then
+    JAVA_TRUSTSTORE="${JAVA_HOME}/lib/security/cacerts"
+else
+    echo "❌ Could not locate Java truststore under JAVA_HOME."
+    exit 1
+fi
+
+echo "📌 Detected Trust Store: $JAVA_TRUSTSTORE"
+
+# Copy truststore to temp p12 file
+TEMP_TRUSTSTORE="temp-cacerts.p12"
```

Comment:
This leaves a load of unstaged junk in the source directory - we should be working in the target directory